### ISSUE TITLE
Issue handling: labels

### ISSUE THREAD

**Issue by EricThomson** [flatironinstitute/ccn-template#12]
Default labels for issues/PRs are not amazing. 

I like to split them up by type/status, but there are other ways too we could discuss. Some repos go overboard. I think pytorch has like 300 labels. But we might want to discuss if we want a convention for labels. 

Also, trying to label, if not reply, within 24 hours (where a weekend counts as 24 hours) goes a long way, I think to telling users "we are listening". PRs especially. Ignoring a PR I see basically as an insult to a user. :frowning_face:

**Comment by billbrod** [comment]
I agree about responding quickly and that better labels are helpful. Any thoughts on what to include? Or a list, or repo to emulate?

**Comment by EricThomson** [comment]
Here's a quick article for brainstorming, it is probably a bit much and more for larger organizational projects:
https://www.fat.codes/articles/tips-issue-labeling-system/

What I use is something like this (note some of these are already part of defaults):

Type:
> type: bug
> type: question  
> type: duplicate
> type: feature request  
> type: documentation
> type: build/install
> type: issue w/another package [e.g., people sometimes will open an issue with caiman that is a problem with numpy]
> type: good first issue


Status (I am less happy with some of these, e.g., we basically never use 'completed' because by that point it is merged, we don't really use 'ready for review' because we just review things and don't tag them when they are ready. So I'd cut some of these. 

> status: awaiting user response 
> status: under development [devs are aware of issue and working on fixing]
> status: under investigation [devs are investigating issue] 
> status: under discussion [ongoing discussion in issue thread]
> status: in progress   [this will apply more to PRs]
> status: awaiting review  [ will apply more to PRs]
> status: completed  [ready for merge: again, applies more to PRs]

**Comment by billbrod** [comment]
Apparently, github can't generate labels from a file in the repo. they can be copied from a [template repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository), which seems like an alternative to cookiecutter and the like.
